### PR TITLE
Add ShowLegend = F to Variant Plots

### PR DESCRIPTION
**Pull request by lfletch0025**

### CODE REVIEW

File: metc-wastewater-covid-monitor/server.R [hunk 1]
```diff
@@ -1,546 +1,548 @@
...
+      mode = "markers",
+      x = ~date,
+      y = ~frequency,
+      split = ~variant,
+      color = ~variant,
+      alpha = 0.8,
+      colors = pal,
+      hoverinfo = "text",
+      text = ~hover_text_variant,
+      showlegend = F
```
Comment (eroten) [inline metc-wastewater-covid-monitor/server.R:295]: This only hides the legend for the markers/points. Is this what we want? How does it look if we hide the legend for the lines?

Comment (lfletch0025) [reply metc-wastewater-covid-monitor/server.R:295]: I can hide it for the lines instead of the markers and do another pull request.  Does that sound good?

Comment (lfletch0025) [reply metc-wastewater-covid-monitor/server.R:295]: Do you want me to change it to markers for both variant plots?

Comment (eroten) [reply metc-wastewater-covid-monitor/server.R:295]: image

Nah, I think this looks good!

**Review by eroten — APPROVED** [review]

**Pull request merged**